addon/-private/options.js

```diff
@@ -18,11 +17,6 @@ export default class Options {
     const optionKeys = keys(options);
     const createParams = { [OPTION_KEYS]: optionKeys, model, attribute };
 
-    // If any of the options is a CP, we need to create a custom class for it
-    if (optionKeys.some(key => isDescriptor(options[key]))) {
```

Comment:
Is this change necessary if the `isDescriptor` logic is working?

Reply:
Good point, thanks. I'll add it back